# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/PDocAccessible.ipdl
# Commit: 16b49e8c315b
# Full Hash: 16b49e8c315bb512c0bc406670d3b4be653185b4
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r 5a6669e44c53 -r 16b49e8c315b accessible/ipc/PDocAccessible.ipdl
--- a/accessible/ipc/PDocAccessible.ipdl	Wed Nov 29 03:53:20 2023 +0000
+++ b/accessible/ipc/PDocAccessible.ipdl	Wed Nov 29 03:53:20 2023 +0000
@@ -70,11 +70,6 @@
   async SelectionEvent(uint64_t aID, uint64_t aWidgetID, uint32_t aType);
   async RoleChangedEvent(role aRole, uint8_t aRoleMapEntryIndex);
   async FocusEvent(uint64_t aID, LayoutDeviceIntRect aCaretRect);
-  async VirtualCursorChangeEvent(uint64_t aID,
-                                 uint64_t aOldPosition,
-                                 uint64_t aPosition,
-                                 int16_t aReason,
-                                 bool aFromUservcEvent);
   async ScrollingEvent(uint64_t aID, uint64_t aType,
                        uint32_t aScrollX, uint32_t aScrollY,
                        uint32_t aMaxScrollX, uint32_t aMaxScrollY);